### PR TITLE
chore(css): round box comment sections

### PR DESCRIPTION
**Pull request by lwander**
I'll be breaking these out by warning, comment, etc... later to allow us
to insert "Warning! Alpha feature..." for all k8s v2 pages

makecommentsnicer

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: _sass/_page.scss [hunk 1]
```diff
@@ -61,6 +61,12 @@
     font-size: 0.8rem;
   }
 
+  blockquote {
+    p {
+      margin: 1em 1em 1em;
```
Comment (ghost) [inline _sass/_page.scss]: v minor nit: shorthand `margin: 1em;` should have the same effect here.

Comment (lwander) [reply _sass/_page.scss]: Isn't that the same as `margin: 1em 1em 1em 1em`? I don't want the left margin to be `1em`

Comment (ghost) [reply _sass/_page.scss]: `margin: 1em 1em 1em;` is also the same as `margin: 1em 1em 1em 1em;`. when you drop the left margin value it takes the value from the right margin unless you override it elsewhere.

Comment (lwander) [reply _sass/_page.scss]: woah

**Pull request merged**